includes/utils.inc

```diff
@@ -202,8 +202,11 @@ function wf_crm_get_events($reg_options, $context) {
     $date_future = date('Y-m-d H:i:s', strtotime($date_future));
     $sql .= " AND (end_date <= '$date_future' OR end_date IS NULL)";
   }
-  if (is_numeric($reg_options['event_type'])) {
-    $sql .= ' AND event_type_id = ' . $reg_options['event_type'];
+  if (is_array($reg_options['event_type'])) {
```

Comment:
Since there are existing webforms which did not save this data as an array, we need to fall-back. I suggest casting it as an array then continuing with your new code.

Reply:
Ok, changed